sphinx/source/docs/dev_guide/bokehjs.rst

```diff
@@ -121,30 +121,45 @@ Use ``node make help`` to list all available commands.
 You can use ``tsc`` directly for error checking (e.g. in an IDE). However, don't use
 it for code emit, because we rely on AST transforms to produce viable library code.
 
+.. _devguide_bokehjs_development_testing:
+
 Testing
 ~~~~~~~
 
-BokehJS testing is performed with the ``node make test`` command. You can run individual
-test suites with ``node make test:suite_name``. Known tests suites are:
+BokehJS comes with its own suites of tests. All tests for BokehJS use ``describe()`` and
+``it()`` functions. Unit tests are written using Chai "expect" style. [TBD: Visual integration
+tests use ``await()`` instead of ``expect()``.]
```

Comment:
This is not accurate. Unit tests can be asynchronous as well (bokehjs is by nature asynchronous), thus you can use e.g. `await display(..)` there. Integration/Visual tests can use `expect()`, but do that very rarely (more often they use `assert()` as it acts as a type guard). In fact, unit and integration differ only in the setup, everything else can be used interchangeably.

Reply:
Thanks, I'll try to make that clearer! How would you describe the style for visual tests - is that 'Chai "expect" style' as well even though most of them don't use ``èxpect()``? What would you say are the differences (in setup) between the unit and integration tests in this case?